src/pykeen/experiments/nodepiece/galkin2022_nodepiece_codexl.yaml

```diff
@@ -0,0 +1,43 @@
+metadata:
+  title: Learn CoDEx-Large Dataset with NodePiece + RotatE as described by Galkin et al., 2022
+pipeline:
+  dataset: CoDExLarge
+  evaluator_kwargs:
+    filtered: true
+  loss: BCELoss
+  model: NodePiece
+  model_kwargs:
+    entity_initializer: xavier_uniform_
+    relation_initializer: init_phases
+    relation_constrainer: complex_normalize
+    embedding_dim: 200   # for both anchors and relations
+    interaction: RotatEInteraction
+    aggregation: mlp
+    tokenizers: ['AnchorTokenizer', 'RelationTokenizer']
+    num_tokens: [20, 6]  # 20 anchors, 6 relations
+    tokenizers_kwargs:
+      [{
+         selection: MixtureAnchorSelection
+         searcher: CSGraphAnchorSearcher
+         selection_kwargs:
+           selections: ['Degree', 'PageRank', 'Random']
+           ratios: [0.4, 0.4, 0.2]
+           num_anchors: 7000
+      },
+       {}  # no params for RelationTokenizer
+      ]
+
+  optimizer: Adam
+  optimizer_kwargs:
+    lr: 0.0005
+  training_kwargs:
+    batch_size: 256
+    num_epochs: 121
+    label_smoothing: 0.3
+  training_loop: LCWA
+results:
```

Comment:
maybe you can add a comment with a (persistent) link to the paper's PDF and the number of the table and location therein

Reply:
done in https://github.com/pykeen/pykeen/pull/771/commits/474e64b064615e48e391271e303fc37a220e07ae